### PR TITLE
WIP: Add support for additional IBBs

### PR DESCRIPTION
**Pull request by ChriMarMe**
Signed-off-by: Christopher Meis <christopher.meis@9elements.com>

### CODE REVIEW

File: cmd/cbnt-prov/cmd.go [hunk 1]
```diff
@@ -383,6 +384,8 @@ func (g *generateKMCmd) Run(ctx *context) error {
 
 func (g *generateBPMCmd) Run(ctx *context) error {
 	var options *cbnt.Options
+	var err error
+	ibbcount := 3
```
Comment (walterchris) [inline cmd/cbnt-prov/cmd.go:388]: Add a constant here like HASH_DGST_PER_IBB or so.

File: cmd/cbnt-prov/cmd.go [hunk 2]
```diff
@@ -406,18 +409,46 @@ func (g *generateBPMCmd) Run(ctx *context) error {
 		se.DMAProtBase1 = g.DMABase1
 		se.DMAProtLimit1 = g.DMASize1
 		se.IBBEntryPoint = g.EntryPoint
+		var fsptIBB, verIBB *bootpolicy.IBBSegment
+		if g.Coreboot {
+			fsptIBB, err = cbnt.FindAdditionalIBBs(g.BIOS, "fspt.bin")
```
Comment (walterchris) [inline cmd/cbnt-prov/cmd.go:414]: Can we make a slice containing the "addition IBBs" and iterate over this? This get's messy if we add more optional IBBs.

**Review by walterchris — CHANGES_REQUESTED** [review]

File: cmd/cbnt-prov/cmd.go (hunk 1 above)
Comment (ChriMarMe) [reply cmd/cbnt-prov/cmd.go]: Done.

File: cmd/cbnt-prov/cmd.go (hunk 2 above)
Comment (ChriMarMe) [reply cmd/cbnt-prov/cmd.go]: done

**Comment by ChriMarMe** [comment]
This is still WIP

**Pull request closed without merging**